force-app/main/default/chat/lwc/chat/chat.js

```diff
@@ -0,0 +1,85 @@
+//todo:: const labels in html
+import {LightningElement, track, wire} from 'lwc';
+import {createRecord, getRecord} from "lightning/uiRecordApi";
+import {subscribe, unsubscribe} from 'lightning/empApi';
+
+import notifyAboutMessageApex from '@salesforce/apex/ChatController.notifyAboutMessage';
+
+import MESSAGE_OBJECT from "@salesforce/schema/Message__c";
+import BODY_FIELD from "@salesforce/schema/Message__c.Body__c";
+
+import MESSAGE_ID_FIELD from "@salesforce/schema/Chat_Event__e.Message_Id__c";
+
+import {MessageWrapperBuilder} from "c/messageItem";
+
+export default class Chat extends LightningElement {
+    /**
+     * @type {MessageWrapper[]}
+     */
+    @track messages = [];
+    receivedMessageId = '';
+    messageFields = ['Message__c.Id', 'Message__c.Body__c', 'Message__c.User_Alias__c', 'Message__c.CreatedDate'];
```

Comment:
This looks like a good candidate for a constant

Reply:
fixed 8d870610e229cc8a420aa5daeddcdb7010ca08a5